infra/auth/main.tf

```diff
@@ -124,5 +124,6 @@ AUTH_URL="${var.AUTH_URL}"
 OPENID_PROVIDER="${var.OPENID_PROVIDER}"
 OIDC_EMAIL_CLAIM="${var.OIDC_AUDIENCE}email"
 OIDC_GROUP_CLAIM="${var.OIDC_AUDIENCE}group"
+OIDC_AUTH0_TOKEN_CLAIM="${var.OIDC_AUDIENCE}auth0"
```

Comment:
cool. just to make sure I understand this `dss_env_vars` section - when these env vars are printed, the operator must manually put them into the data store `environment` file to update them, correct?

Reply:
correct, this and the `internal/application_secrets.json` are provided so they can use them within the data-store.